src/test/java/com/google/api/codegen/testdata/py/python_library.baseline

```diff
@@ -4107,7 +4107,6 @@ names = [names_element]
 shelves_element = 'Classics'
 shelves = [shelves_element]
 
-# Iterate over all results
```

Comment:
Given the level of commenting in the published samples, I would keep these comments in there.

I don't feel strongly about this, though, since we'll be getting feedback on the samples anyway.

Reply:
I'll revert.